### PR TITLE
forward os signals to go test

### LINKED ISSUE #14: os signals are not passed on to go test process

### PR DESCRIPTION
**Pull request by frzifus**
Closes #14

### CODE REVIEW

**Comment by subtlepseudonym** [comment]
If you open a copy of this PR against [subtlepseudonym/gotest](https://github.com/subtlepseudonym/gotest), I'll merge it.

File: main.go [hunk 1]
```diff
@@ -49,6 +50,24 @@ func gotest(args []string) int {
...
 
+	sigc := make(chan os.Signal)
+	done := make(chan struct{})
+	defer func() {
+		done <- struct{}{}
+	}()
+	signal.Notify(sigc)
+
+	go func() {
+		for {
```
Comment (rakyll) [inline main.go:61]: You don't need the for loop, right? Just listening on sigc would be sufficient? It's gonna block and forward the signal if something happens. Am I wrong?

Comment (subtlepseudonym) [reply main.go:61]: My understanding is that this will forward multiple signals whereas omitting the for loop would only forward the first signal.

Comment (frzifus) [reply main.go:61]: yes, exactly. for example, i could send `SIGUSR1` and if the program doesn't react, `SIGKILL` later.

Comment (rakyll) [reply main.go:61]: Gotcha, thanks.

**Pull request merged**